nitrite.go

```diff
@@ -342,20 +362,28 @@ func Verify(data []byte, options VerifyOptions) (*Result, error) {
 	if nil == roots {
 		roots = defaultRoot
 	}
+	if cert.IsCA {
```

Comment:
If the certificate is a CA certificates (as it the case for a `nitriding` `StandaloneAttester`) we need to add the certificates to `roots`, so that `cert.Verify` online 374 passes.